dev/inc/CppWinRTHelpers.h

```diff
@@ -160,9 +158,9 @@ struct PropertyChanged_revoker
        return *this;
    }
 
-   PropertyChanged_revoker(winrt::DependencyObject const& object, winrt::DependencyProperty const& dp, int64_t token) :
+   PropertyChanged_revoker(winrt::DependencyObject const& object, const winrt::DependencyProperty&  dp, int64_t token) :
         m_object(object),
-        m_property(dp),
+        m_property(std::move(dp)),
         m_token(token)
```

Comment:
Which tidy rule applied here to put std::move? I'm still trying to understand the reasons. :)